workflow/sb_multicnv_wf.cwl

```diff
@@ -0,0 +1,331 @@
+    sbg:fileTypes: CSV
+  combine_stats_total:
+    type: File[]?
+    outputSource: add_basename_combine_stats_total/output
+    label: Total statistics
+    doc: |-
+      Total length of genome per each combination of statuses / status ('gain', 'neutral', 'loss').
+    sbg:fileTypes: CSV
+  combine_status_CN_table_overview:
+    type: File?
+    outputSource: add_basename_combine_status_CN_table_overview/output
+    label: Combined regions - CN table
+    doc: Resulted .CSV file which present combined regions with copy number values
+    sbg:fileTypes: CSV
+  combine_status_CR_table_overview:
+    type: File?
+    outputSource: add_basename_combine_status_CR_table_overview/output
+    label: Combined regions - CR table
+    doc: Resulted .CSV file which present combined regions with copy ratio values
+    sbg:fileTypes: CSV
+  combine_status_table_overview:
+    type: File?
+    outputSource: add_basename_combine_status_table_overview/output
+    label: Combine regions - status table
+    doc: Resulted .CSV file which present combined regions with status values
+    sbg:fileTypes: CSV
+  combine_uniformed_view:
+    type: File[]?
+    outputSource: add_basename_combine_uniformed_view/output
+    label: Uniformed view per file
+    doc: Uniformed view per file with combined regions.
+    sbg:fileTypes: CSV
+  html_visualisations:
+    type: File[]?
+    outputSource: add_basename_html_visualisations/output
+    label: Interactive visualisations
+    doc: |-
+      Interactive visualisations presenting:
+      1. combined calls
+      2. sum of lengths
+      3. total sum of lengths
+    sbg:fileTypes: HTML
+  interpret_majority:
+    type: File?
+    outputSource: add_basename_interpret_majority/output
+    label: Result for 'majority' interpretation
+    doc: |-
+      Resulting .CSV file produced after 'majority' interpretation method is used to combine regions.
+    sbg:fileTypes: CSV
+  interpret_precise:
+    type: File?
+    outputSource: add_basename_interpret_precise/output
+    label: Result for 'precise' interpretation
+    doc: |-
+      Resulting .CSV file produced after 'precise' interpretation method is used to combine regions.
+    sbg:fileTypes: CSV
+  ploidy_information:
+    type: File?
+    outputSource: add_basename_ploidy_information/output
+    label: Ploidy Information
+    doc: Information about the ploidy of different callers / tumors.
+    sbg:fileTypes: JSON
+  segment_result_final:
+    type: File[]?
+    outputSource: add_basename_segment_result_final/output
+    label: Result after segmentation
+    doc: |-
+      Resulted file after merging the regions with the same final status excluding 'ambiguous' regions'
+    sbg:fileTypes: CSV
+  segment_results_with_ambiguous:
+    type: File[]?
+    outputSource: add_basename_segment_results_with_ambiguous/output
+    label: Result after segmentation with ambiguous
+    doc: Resulted file after merging the regions with the same final status
+    sbg:fileTypes: CSV
```

Comment:
Hmmm, does `sbg:fileTypes` need to be quoted to work with sbpack?

Reply:
Weirdly enough, it worked fine. Must be more forgiving when it is in the yaml format.

Reply:
ooooh, interesting